pallas-applying/Cargo.toml

```diff
@@ -13,8 +13,14 @@ authors = ["Maico Leberle <maico.leberle@gmail.com>"]
 doctest = false
 
 [dependencies]
+cryptoxide = "0.1"
+cbor_event = "^2.1.1"
```

Comment:
we already have a crate for dealing with cbor serialization, it's called `minicbor`. You can access it via `pallas::codec::minicbor`. Any particular reason we need `cbor_event`?

Reply:
I used `cbor_event` trying to mimick what's done in [rust-byron-cardano](https://github.com/input-output-hk/rust-byron-cardano/blob/5f0c8d34424e0ccb199b8c6b70fd410d4bd6af41/cardano/src/tx.rs#L148), specifically the part concerned with preparing the bytes array on which the witness signatures are verified. I can try using minicbor instead, and see if I can still check witnesses

Reply:
I was just trying to copy the implementation in [rust-byron-cardano](https://github.com/input-output-hk/rust-byron-cardano/blob/5f0c8d34424e0ccb199b8c6b70fd410d4bd6af41/cardano/src/tx.rs#L148). But nevermind, I just added a [commit](cf3067f6eb3e362e24d2f19e31db204cda92564e) changing it `minicbor` instead.